JitsiMediaDevices.js

```diff
@@ -156,7 +156,19 @@ class JitsiMediaDevices {
                 }
 
                 Promise.all(promises).then(
-                    r => resolve(r.every(Boolean)),
+                    r => {
```

Comment:
what is `r` ?  is `response ` better here ?

Reply:
response shortened as r is a generally used convention